package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "lint-staged",
   "description": "Lint files staged by git",
-  "main": "index.js",
-  "bin": {
-    "lint-staged": "index.js"
-  },
+  "license": "MIT",
+  "repository": "https://github.com/okonet/lint-staged",
+  "author": "Andrey Okonetchnikov <andrey@okonet.ru>",
```

Comment:
Please add maintainers section with you and @sudo-suhas included. BTW any way to automate that taking this from GH API somehow?

Reply:
yeah it's possible, let me take a look at some options

Reply:
hmm after reading the docs, it's only possible for orgs

Reply:
Too bad. I might consider moving lint-staged to a separate org in the future so let's postpone this for now. I'd still like to see you and @sudo-suhas as maintainers.

Reply:
for now we can only do this manually